### ISSUE TITLE
ARIA in HTML conformance to conform WCAG ?

### ISSUE THREAD

**Issue by goetsu** [w3c/wcag#717]
Hi,

I would like to have the WG opinion regarding status of ARIA in HTML regarding conformance to WCAG 
Is there any SC that will require to fully conform to ARIA in HTML https://www.w3.org/TR/html-aria/ document to be considered compliant with WCAG ? or is it like with SC 4.1.1 and HTML conformance only some part of this specs that need to be considered ? or no obligation at all ?

**Comment by patrickhlauke** [comment]
i don't believe there's any hard requirement to fully conform to the ARIA in HTML document (which is only a working draft at this stage as well). however, going against that document will likely lead to actual WCAG failures in 4.1.1, 4.1.2, 4.1.3.

i'd say this has parallels with the other discussion about 4.1.1 in general (and how, strictly speaking -though there's debate around how strictly - there's no requirement to have completely valid HTML documents) https://github.com/w3c/wcag/issues/542

**Comment by mraccess77** [comment]
Personally full ARIA validation does not fall under 4.1.1.  Most validation issues affecting users would likely fall under 4.1.2 or possibly 1.3.1 or 4.1.3.   There are some ARIA constructs that I would require -- partial validation -- that is when you use X you must also use Y.  That is there are certain constructs that must be implemented in a certain way to work with assistive technology and there are certain combinations of attributes that work together with roles as well that are needed and should be validated because they don't apply to certain roles and would not be seen by AT or could cause access issues.  So we have created a limited set of ARIA validation rules for automated testing.

**Comment by stevefaulkner** [comment]
@patrickhlauke 
> (which is only a working draft at this stage as well)
**note:** this does not effect the normativity of the ARIA in HTML requirements as expressed in the w3c HTML validator

**Comment by goetsu** [comment]
So look more to be a case by case basis than full conformance to the document required

**Comment by jake-abma** [comment]
> Is there any SC that will require to fully conform to ARIA in HTML

Answer: no

:-)

**Comment by awkawk** [comment]
Proposed WG Response:
The discussion on this issue captures the essence of the WG view on this question - that WCAG does not require to fully conform to the ARIA in HTML document. However, going against that document will likely lead to actual WCAG failures different SC.

**Comment by alastc** [comment]
The WG accepted the proposed response, closing.
